app/src/main/java/com/cobresun/factfulnewsandroid/ui/activities/MainActivity.kt

```diff
@@ -17,6 +22,11 @@ class MainActivity : AppCompatActivity() {
         val toolbar = findViewById<Toolbar>(R.id.toolbar)
         setSupportActionBar(toolbar)
 
+        AppCenter.start(
+            application, "40e7957a-3008-49b7-af7b-5bf1be103ad5",
```

Comment:
This magic number would do well being defined explicitly or even hidden possibly?

Reply:
Good idea